packages/context/README.md

```diff
@@ -133,6 +133,30 @@ export class MyApp extends LitElement {
 }
 ```
 
+`ContextProvider` can also be used with plain HTML elements. This can be
+useful to provide a context provider without introducing a custom element:
+
+#### **`my-app.js`**:
+
+```js
+import {ContextProvider} from '@lit/context';
+import {loggerContext, Logger} from './logger.js';
+
+// create a provider for the whole document body.
+const loggingProvider = new ContextProvider(document.body, {
+    context: loggerContext,
+    initialValue: {
+      log: (msg) => {
+        console.log(`[global] ${msg}`);
+      },
+    },
+);
+```
+
+If the provider is being added when there is already a consumer registered with
+a parent of the specified element or with a `ContextRoot`, then
+`.hostConnected()` must be called on the provider after creating it.
```

Comment:
Nit: Should there be a slight addition of why it must be called.
E.g., to ensure any consumers are updated to use their nearest provider correctly?

[non blocking comment]

Reply:
Yeah I was debating if we should add some more there. Good call out.